### PR TITLE
Add quilt3 login FAQ

### PR DESCRIPTION
**Pull request by robnewman**
# Description
Add `quilt3 login` to help users manage Quilt from containers like Docker

# TODO

- [ x ] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -125,3 +125,14 @@ for (k, e) in p.walk():
 
 You can then follow the above with `q3.Package.delete(pname, registry=reg, top_hash=p.top_hash)`.
 
+## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?
+
+If you have configured your [AWS command line](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
+you can run `aws configure` and use your valid AWS credentials to push from your
+machine or from any pipeline. Remember to run `quilt3 logout` if you were logged 
+in to your existing Quilt stack!
```
Comment (akarve) [inline docs/FAQ.md:133]:
```suggestion
Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
and `quilt3` will use the same for its API calls. Select among multiple profiles in your shell as follows:
```bash
export AWS_PROFILE=your_profile
```

> Be sure to `quilt3 logout` if you were previously logged in to your existing Quilt stack and wish to use
AWS credentials instead.

File: docs/FAQ.md [hunk 2]
```diff
@@ -125,3 +125,14 @@ for (k, e) in p.walk():
...
+
+The necessary AWS S3 object permissions are similar to 
+[this bucket policy](https://docs.quiltdata.com/advanced/crossaccount#bucket-policies) 
+but you do not need either the `s3:GetBucketNotification` or `s3:PutBucketNotification` 
+to use `quilt3`.
```
Comment (akarve) [inline docs/FAQ.md:138]:
```suggestion
The S3 permissions needed by `quilt3` are similar to
[this bucket policy](https://docs.quiltdata.com/advanced/crossaccount#bucket-policies)
but `quilt3` does not need either `s3:GetBucketNotification` or `s3:PutBucketNotification`.
```

**Review by akarve — CHANGES_REQUESTED** [review]

**Comment by drernie** [comment]
I defer to @akarve

**Comment by robnewman** [comment]
@akarve Suggested changes incorporated. Merged with latest `master` branch

**Review by akarve — DISMISSED** [review]

**Comment by robnewman** [comment]
Had to remerge with latest master, so needs another review

File: docs/FAQ.md [hunk 3]
```diff
@@ -125,6 +125,18 @@ for (k, e) in p.walk():
 
 You can then follow the above with `q3.delete_package(pname, registry=reg, top_hash=p.top_hash)`.
 
+## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?
+
+Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
+and `quilt3` will use the same for its API calls. Select among multiple profiles in your shell as follows:
+```bash
+export AWS_PROFILE=your_profile
+```
```
Comment (akarve) [inline docs/FAQ.md:134]: Add a note on "Be sure to run `quilt3 logout` if you've previously logged in."

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -70,6 +70,7 @@
 * [Changed] Update Okta documentation and examples ([#3126](https://github.com/quiltdata/quilt/pull/3126))
 * [Added] Add Amazon Athena documentation, examples and references ([#3116](https://github.com/quiltdata/quilt/pull/3116))
 * [Fixed] Fix four-deep headers so auto-link generation works ([#3100](https://github.com/quiltdata/quilt/pull/3100))
+* [Added] Using quilt3 login and aws configure with containers ([#3099](https://github.com/quiltdata/quilt/pull/3099))
```
Comment (akarve) [inline docs/CHANGELOG.md:73]: I don't think that we should changelog docs additions. That's really confusing. Added is for features.

File: docs/FAQ.md (hunk 3 above)
Comment (robnewman) [reply docs/FAQ.md:139]: Added

File: docs/CHANGELOG.md (hunk 4 above)
Comment (robnewman) [reply docs/CHANGELOG.md]: Based on review of our company [changelog design guide](https://keepachangelog.com/en/1.0.0/), removed all docs from changelog

File: docs/FAQ.md [hunk 5]
```diff
@@ -125,6 +126,22 @@ for (k, e) in p.walk():
 
 You can then follow the above with `q3.delete_package(pname, registry=reg, top_hash=p.top_hash)`.
 
+## Do I have to login via quilt3 to use the Quilt APIs? How do I push to Quilt from a headless environment like a Docker container?
+
+Configure [AWS CLI credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) 
+and `quilt3` will use the same for its API calls.
+
+Note: be sure to run `quilt3 logout` if you've previously logged in.
```
Comment (akarve) [inline docs/FAQ.md:134]:
```suggestion
> Be sure to run `quilt3 logout` if you've previously logged in.
```

**Comment by robnewman** [comment]
Abandoning branch. These changes were applied in another branch

**Pull request closed without merging**